sotodlib/tod_ops/fft_ops.py

```diff
@@ -353,6 +390,8 @@ def fit_noise_model(
     f_max=100,
     merge_name="noise_fit_stats",
     merge_psd=True,
+    freq_spacing=None,
+    approx_fit=False,
```

Comment:
Sounds good.
It's a little confusing that `calc_wn` returns the square root of `median(psd)` whereas `fit_noise_model` doesn't take the root; do we want to change that? Also seems cleaner to add a new function `estimate_fknee` but I can mash it all into `calc_wn` if you prefer

Reply:
Yes, this bothers me quite a lot that we (probably I) decided to change the convention between `calc_wn` and `fit_noise_model`. I think that they should both be in squared units (like `fit_noise_model` is now). I see what you're saying about wanting to have another function like `estimate_fknee` but I think my preference is to have one function called `estimate_noise_model` which returns exactly the same shaped output as `fit_noise_model` i.e. an axis shaped (dets x noise_coeffs). If we want to keep `calc_wn` fn alive for backwards compatibility we could just have it call `estimate_noise_model` (so that we don't have to update definitions of median/mean/etc in multiple places) and just return the white noise coefficients but I would keep everything in the squared units for consistency.

Another related issue was brought up by @ykyohei in https://github.com/simonsobs/sotodlib/issues/995 where we really should be always taking the PSDs w/ overlap = 0, storing the nperseg in the calc_psd output so that we can have the calc_wn/any fn that takes a median over psd bins correct for this "median bias". 

It's maybe ok to address that all here but also ok opening up a new PR for these fixes and reviewing that separately, I'll leave it up to you.

Reply:
After some consideration I think `estimate_noise_model` either needs to be more robust then just the initial guess of the `fit_noise_model` or at least have some error quantification. Otherwise you can have catastrophic mis-fits  -- this is also a problem with `fit_noise_model` but I assume it could be caught with the covariance. I've just deleted `approx_fit` for now, will open a new PR (or at least an issue) to deal with these other problems.